pytorch/buildspec.yml

```diff
@@ -2,7 +2,7 @@
   region: &REGION <set-$REGION-in-environment>
   framework: &FRAMEWORK pytorch
   version: &VERSION 1.6.0
-  cuda_version: &CUDA_VERSION cu101
+  cuda_version: &CUDA_VERSION cu110
   os_version: &OS_VERSION ubuntu16.04
```

Comment:
[nit] Can you add a buildspec-1-6-0-cu101.yml (or something of the likes) that has a copy of the old 1.6.0 CUDA 10.1 buildspec file? It turns out they're useful to have readily available for bug fixes, etc.

Reply:
Sounds good, addressed these changes.